### ISSUE TITLE
macOS Client - 1.16.4 is not signed correctly

### ISSUE THREAD

**Issue by tonyb486** [bitwarden/desktop#312]
When I try and run 1.16.4, it says:
> “Bitwarden 1.16.4.app” can’t be opened because it is from an unidentified developer.

Trying to verify the signature status shows:
```
unexpected ➜  ~ spctl -a -t exec -vv /Applications/Bitwarden\ 1.16.4.app
/Applications/Bitwarden 1.16.4.app: rejected
origin=Developer ID Application: 8bit Solutions LLC (LTZ2PFU5D6)
unexpected ➜  ~ spctl -a -t exec -vv /Applications/Bitwarden\ 1.16.2.app
/Applications/Bitwarden 1.16.2.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: 8bit Solutions LLC (LTZ2PFU5D6)
```

I'm not sure what's wrong with how it's signed, or if my system is somehow configured wrong. The sha512sum seems to match what's in the yaml file, so I don't think it's corrupt (and I think spctl would tell you if that's the reason why the signature failed).

**Comment by kspearrin** [comment]
Looking into it. Thanks.

**Comment by kspearrin** [comment]
Should be fixed with 1.16.6. Available on website now. Please let me know.

**Comment by tonyb486** [comment]
Works for me now, thanks!
